src/log.js

```diff
@@ -75,7 +75,7 @@ export const LogLevel = {
 /**
  * Sets reportError function. Called from error.js to break cyclic
  * dependency.
- * @param {function(*, !Element=)|undefined} fn
+ * @param {function(this:Window, Error, (Element|null)=): ?|undefined} fn
```

Comment:
This file is an area of active work/changes, if it's possible to exclude it for now that would be ideal as it's going to change a lot in the near future. If not possible because of deps, oh well

Reply:
Not possible :/

Reply:
![Screen Shot 2021-04-29 at 11 59 05 AM](https://user-images.githubusercontent.com/4656974/116581430-5231c200-a8e2-11eb-93fd-13e95645dbd0.png)